src/index.html

```diff
@@ -7,8 +7,53 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link 
+      rel="preconnect" 
+      href="https://fonts.googleapis.com">
+    <link 
+      rel="preconnect" 
+      href="https://fonts.gstatic.com">
+    <link 
+    href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" 
+    rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#home">
+        <img
+        src="./images/logo.png" 
+        alt="logo" 
+        class="logo"
+        >
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#" class="nav__link is-active">apple</a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">samsung</a>
+          </li>
```

Comment:
Apply such code styling recommendations all over the document
```suggestion
        <ul class="nav__list">
          <li class="nav__item">
            <a href="#" class="nav__link is-active">
              apple
            </a>
          </li>
          
          <li class="nav__item">
            <a href="#" class="nav__link">
              samsung
            </a>
          </li>
```